common/cache.py

```
@@ -0,0 +1,190 @@
+from common.metrics import metrics
+
+
+class CachedTemplateView(TemplateView):
+    @classmethod
+    def as_view(cls, **initkwargs):  # @NoSelf
+        return cache_page(settings.CACHE_TTL)(
+            super(CachedTemplateView, cls).as_view(**initkwargs)
+        )
+
+
+class NeverCacheMixin(object):
```

Ruff (0.12.2)

Class `NeverCacheMixin` inherits from `object`

Remove `object` inheritance

(UP004)